pkg/bootstrap/docker/openshift/metrics.go

```diff
@@ -186,7 +187,27 @@ func metricsDeployerPod(hostName, imagePrefix, imageVersion string) *kapi.Pod {
 			Env: env,
 		},
 	}
-	return pod
+
```

Comment:
You really don't need a pod object anymore (https://github.com/openshift/origin/pull/12174/files#diff-8f476646dd3c6f6850aa06d1673053d7L147)

you should just create a pod spec and directly use it below